src/client/entity-editor/edition-section/edition-section.js

```diff
@@ -271,6 +271,7 @@ function EditionSection({
 								<Button
 									block
 									bsStyle="primary"
+									id="existing-edition-group-search-button"
```

Comment:
I'm not sure about the use of an id here.
It would be much better if the solution was generic and applied to other buttons that could potentially have that issue

Reply:
@MonkeyDo 
i have possibly checked whole website , i didn't
 find this bug anywhere else , that's why i did'nt stressed much about generic solution at the first place .